ConnectionsPanel.js

```diff
@@ -420,4 +382,110 @@ ToolsButton.propTypes = {
   count:         PropTypes.number,
 };
 
+const ConnectionsPanelSection = ({ title, children }) => {
+  const { themeStr, interfaceLanguage } = useContext(GlobalStateContext);
+  const theme = getTheme(themeStr);
+  return (
+    <View style={styles.connectionPanelSection}  flexdir={interfaceLanguage === "english" ? null : styles.rtlRow}>
```

Comment:
I'm pretty sure `flexdir` isn't a valid prop on `View`. Is this logic working? Use `<FlexFrame/>` in Misc.js instead.